### ISSUE TITLE
Changing default max_epoch training time

### ISSUE THREAD

**Issue by moiseshorta** [caillonantoine/msprior#9]
Hello!

I have recently been training an MSPrior module (decoder_only and recurrent), and have noticed that the
training process stops at 1000 epochs by default. 
I think this is too little time to train a model, as the loss seems to keep going down
when I resume it, but immediately stops as it is going beyond the 1000 epochs.

Would it be possible to change this param?, it would be valuable to have it exposed in the CLI.

All the best,
M

**Comment by moiseshorta** [comment]
Just realized the problem was being caused by the `
        pl.callbacks.EarlyStopping(
           "val_cross_entropy",
           patience=20,)`

in the `train.py` script. I've just commented it out and it seems the loss still continues to go down, for those wanting to extend the training.

**Comment by futile-device** [comment]
Not totally sure but I think changing the max epochs could maybe happen when calling the trainer?

I found adding `max_epochs=-1` to the pl.Trainer call at the end of `train.py` works, ie:

`    trainer = pl.Trainer(
        logger=pl.loggers.TensorBoardLogger('runs', name=FLAGS.name),
        accelerator='gpu',
        devices=[FLAGS.gpu],
        callbacks=callbacks,
        log_every_n_steps=10,
        max_epochs=-1,
        **val_check,
    )`
